apack.json

```diff
@@ -98,16 +99,16 @@
       "cmd": "$(zap) server --reuseZapInstance --studioHttpPort ${studioHttpPort}"
     },
     "zapFull": {
-      "cmd": "$(zap) --showUrl --zcl ${sdkRoot}/app/zcl/zcl-zap.json"
+      "cmd": "$(zap) --showUrl --zcl ${sdkRoot}/app/zcl/zcl-zap.json [ --zcl ${zcl.zigbeeZclJsonFile} ]"
```

Comment:
I'm confused with these..... Is it `sdkRoot` or is it `zcl.zigbeeZclJsonFile` ?

If this is "one or the other, then why is the WHOLE thing not in square brackets, like:
`
[ --zcl ${sdkRoot}/app/zcl/zcl-zap.json | --zcl ${zcl.zigbeeZclJsonFile} ]
`